components/restservice/scripts/install_plugins.sh

```diff
@@ -7,20 +7,24 @@ echo "Installing rest service plugins"
 
 {% for name, plugin in plugins.iteritems() %}
 
-echo "Installing {{ name }} plugin in rest service virtual environment"
+    echo "Installing {{ name }} plugin in rest service virtual environment"
 
-extract_plugin_script="\
-import sys;\
-from cloudify_agent.api.plugins.installer import extract_package_to_dir;\
-sys.stdout.write(extract_package_to_dir('{{ plugin.source }}'));\
-"
+    extract_plugin_script="import sys;\
+    from cloudify_agent.api.plugins.installer import extract_package_to_dir;\
+    sys.stdout.write(extract_package_to_dir('{{ plugin.source }}'));\
+    "
 
-plugin_dir=$(python -c "${extract_plugin_script}")
+    plugin_dir=$(python -c "${extract_plugin_script}")
 
-pushd ${plugin_dir}
-sudo $VENV/bin/pip install . {{ plugin.install_args }}
-popd
+    pushd ${plugin_dir}
+        if [[ {{ plugin.source }} == *.wgn ]]; then
+            # install wagon plugin
+            sudo $VENV/bin/pip install {{ plugin.install_args }}
```

Comment:
We do, actually. It's required to use sudo to install dependencies in the manager env